locales/base/translation.json

```diff
@@ -2660,7 +2660,11 @@
       "ageOfPool": "Age of Pool",
       "learnMoreOnProvider": "View Pool on {{providerName}}",
       "deposit": "Deposit",
-      "withdraw": "Withdraw"
+      "depositAndEarnings": "Deposit & Earnings",
+      "withdraw": "Withdraw",
+      "totalDepositAndEarnings": "Total Deposit & Earnings",
+      "titleLocalAmountDisplay": "{{localCurrencySymbol}}{{localCurrencyAmount}}",
+      "lineItemAmountDisplay": "{{localCurrencySymbol}}{{localCurrencyAmount}} ({{cryptoAmount}} {{cryptoSymbol}})"
```

Comment:
these don't seem to have any language specific stuff that needs to be localized. Can we just do this concatenation in code?

Reply:
Currency symbols can be after the amount in other languages, I am not sure if this would be the case for crypto symbols, but I decided to air on the side of caution.